packages/models/proto/v2/notification-config/events.proto

```diff
@@ -4,10 +4,26 @@ package notification.config.v2;
 
 import "v2/notification-config/notification-config.proto";
 
+message TenantNotificationConfigCreatedV2 {
+  TenantNotificationConfigV2 tenantNotificationConfig = 1;
+}
+
 message TenantNotificationConfigUpdatedV2 {
   TenantNotificationConfigV2 tenantNotificationConfig = 1;
 }
 
+message TenantNotificationConfigDeletedV2 {
+  TenantNotificationConfigV2 tenantNotificationConfig = 1;
```

Comment:
not sure if the config is useful to have here in the event, as opposed to just the ID, but I think this is done consistently in V2 events